### PR TITLE
Implement  jax.numpy.round

### LINKED ISSUE #10195: round

### PR DESCRIPTION
**Pull request by manuelsolan-o**
Close #10195

### CODE REVIEW

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by suvadityamuk** [comment]
Hi, @manuelsolan-o! Apologies for the delay :sweat_smile: 

Please fix the current merge conflicts as visible, the linting (by using the [pre-commit hook](https://lets-unify.ai/ivy/contributing/0_setting_up.html#pre-commit)), and make sure that the tests are passing. I'll keep your PR on a priority and make sure I get on this as soon as you finish up making these changes. 

Thank you for taking the time to make the PR! :slightly_smiling_face:

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.
